scripts/ci/sizes.py

```diff
@@ -130,19 +130,19 @@ def compare(previous_path: str, current_path: str, threshold: float) -> None:
                         name,
                         f"{cell(previous, div)} {unit}",
                         f"{cell(current, div)} {unit}",
-                        sign + str(change) + "%",
+                        f"{sign}{change:.2f}%",
                     )
                 )
         elif "current" in entry:
             value = entry["current"]["value"]
             unit = entry["current"]["unit"]
 
-            rows.append((name, "(none)", f"{value} {unit}", "+100%"))
+            rows.append((name, "(none)", f"{value:.2f} {unit}", "+100%"))
```

Comment:
The values are already stored as strings rounded up to 2 decimal places
```suggestion
            rows.append((name, "(none)", f"{value} {unit}", "+100%"))
```